<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>869</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>5</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CompassTag Data Visualization</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabConfig">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_graph">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4">
        <item>
         <widget class="QWidget" name="plotWindow" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,1">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCustomPlot" name="plot" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>200</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QQuickWidget" name="quickWidget">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>400</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
             </property>
             <property name="resizeMode">
              <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_info">
       <attribute name="title">
        <string>File Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="te_fileinfo"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>24</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionActivity"/>
    <addaction name="actionHeading"/>
    <addaction name="separator"/>
    <addaction name="actionVoltage"/>
    <addaction name="actionTemperature"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="actionCalibration_Constants"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionUTC_Offset"/>
    <addaction name="separator"/>
    <addaction name="actionCompass_Declination"/>
    <addaction name="actionBattery_Forward"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuView"/>
   <addaction name="menuConfiguration"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionActivity_File">
   <property name="text">
    <string>Activity File</string>
   </property>
   <property name="toolTip">
    <string>Read CSV start/stop data and generate activity csv</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionBattery_Direction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Battery Forward</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Zoom Full</string>
   </property>
  </action>
  <action name="actionActivity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Activity</string>
   </property>
  </action>
  <action name="actionVoltage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Voltage</string>
   </property>
  </action>
  <action name="actionTemperature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </action>
  <action name="actionCompass_Declination">
   <property name="text">
    <string>Compass Declination</string>
   </property>
  </action>
  <action name="actionBattery_Forward">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Battery Forward</string>
   </property>
  </action>
  <action name="actionUTC_Offset">
   <property name="text">
    <string>UTC Offset</string>
   </property>
  </action>
  <action name="actionCalibration_Constants">
   <property name="text">
    <string>Calibration Constants</string>
   </property>
  </action>
  <action name="actionHeading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Heading</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
